validator/client/validator.go

```diff
@@ -1242,25 +1241,41 @@ func (v *validator) EventStreamIsRunning() bool {
 	return v.validatorClient.EventStreamIsRunning()
 }
 
-func (v *validator) HealthTracker() health.Tracker {
-	return v.nodeClient.HealthTracker()
-}
-
 func (v *validator) Host() string {
 	return v.validatorClient.Host()
 }
 
 func (v *validator) ChangeHost() {
-	if len(v.beaconNodeHosts) == 1 {
-		log.Infof("Beacon node at %s is not responding, no backup node configured", v.Host())
-		return
-	}
 	next := (v.currentHostIndex + 1) % uint64(len(v.beaconNodeHosts))
-	log.Infof("Beacon node at %s is not responding, switching to %s...", v.beaconNodeHosts[v.currentHostIndex], v.beaconNodeHosts[next])
+	log.WithFields(logrus.Fields{
+		"current_host": v.beaconNodeHosts[v.currentHostIndex],
+		"next_host":    v.beaconNodeHosts[next],
```

Comment:
```suggestion
		"currentHost": v.beaconNodeHosts[v.currentHostIndex],
		"nextHost":    v.beaconNodeHosts[next],
```
We use camel case everywhere else for log fields